pages/linux/urxvt.md

```diff
@@ -0,0 +1,24 @@
+# urxvt
+
+> Rxvt-unicode.
+> A customizable terminal emulator.
+
+- Open a new urxvt window:
+
+`urxvt`
+
+- Run in a specific working directory:
+
+`urxvt -cd {{directory}}`
+
+- Run a command in a new urxvt window:
+
+`urxvt -e {{command}}`
```

Comment:
Is there an expanded version of `e` ? Right now, it does not map to any word.

Reply:
Sadly, there is no expanded version of "-e".